spec/features/person_view_spec.rb

```diff
@@ -31,4 +31,38 @@
     expect(current_path).to eq(person_path(person))
     expect(page).to have_content('123-4567')
   end
+
+  it "has links to edit phone numbers" do
+    person.phone_numbers.each do |phone|
+      expect(page).to have_link('edit', href: edit_phone_number_path(phone))
+    end
+  end
+
+  it "edits a phone number" do
+    phone = person.phone_numbers.first
+    old_number = phone.number
+
+    first(:link, 'edit').click
+    page.fill_in('Number', with: '555-9191')
+    page.click_link_or_button('Update Phone number')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.